toolbox/fdc3-for-web/fdc3-web-impl/src/handlers/HeartbeatHandler.ts

```diff
@@ -95,35 +101,41 @@ export class HeartbeatHandler implements MessageHandler {
     clearInterval(this.timerFunction);
   }
 
-  accept(msg: any, sc: ServerContext<AppRegistration>, from: InstanceID): void {
+  accept(
+    msg: AppRequestMessage | WebConnectionProtocol6Goodbye,
+    sc: ServerContext<AppRegistration>,
+    from: InstanceID
+  ): void {
     if (!this.contexts.includes(sc)) {
       this.contexts.push(sc);
     }
 
     if (msg.type == 'heartbeatAcknowledgementRequest') {
       const app = sc.getInstanceDetails(from);
       if (app) {
-        this.lastHeartbeats.set(app.instanceId!!, new Date().getTime());
+        this.lastHeartbeats.set(app.instanceId, new Date().getTime());
       }
     }
 
-    if (msg.type == 'WCP5Shutdown') {
+    if (msg.type == 'WCP6Goodbye') {
       const app = sc.getInstanceDetails(from);
```

Comment:
this is no longer tested

Reply:
Wasn't tested previously:
![image](https://github.com/user-attachments/assets/13892e83-e5e7-46a9-8e0b-cca98d71a84f)

Will add something for it.